test-framework/maven/src/main/java/io/quarkus/maven/it/continuoustesting/TestModeContinuousTestingMavenTestUtils.java

```diff
@@ -18,10 +22,16 @@ public class TestModeContinuousTestingMavenTestUtils extends ContinuousTestingMa
     // Example output we look for
     // 1 test failed (1 passing, 0 skipped), 1 test was run in 217ms. Tests completed at 21:22:34 due to changes to HelloResource$Blah.class and 1 other files.
     // All 2 tests are passing (0 skipped), 2 tests were run in 1413ms. Tests completed at 21:22:33.
-    private static final Pattern ALL_PASSING = Pattern.compile("All (\\d\\d*) tests are passing \\((\\d\\d*) skipped\\)",
+    // Windows log, despite `quarkus.console.basic=true', might contain terminal control symbols, colour decorations.
+    // e.g. the matcher is then fighting: [39m[91m1 test failed[39m ([32m1 passing[39m, [94m0 skipped[39m)
+    private static final Pattern ALL_PASSING = Pattern.compile(
+            "(?:\\e\\[[\\d;]+m)*All (\\d+) tests are passing \\((\\d+) skipped\\)",
             Pattern.MULTILINE);
     private static final Pattern SOME_PASSING = Pattern
-            .compile("(\\d\\d*) tests? failed \\((\\d\\d*) passing, (\\d\\d*) skipped\\)", Pattern.MULTILINE);
+            .compile(
```

Comment:
I would rather stripping of the ansi characters so that it can work with the original regexp which is something I can follow.